site/app/libraries/database/DatabaseQueries.php

```diff
@@ -6559,6 +6559,11 @@ public function getTimeJoinedQueue($user_id, $queue_code) {
         return $this->course_db->rows()[0]['max'];
     }
 
+    public function getStarType($user_id, $queue_code) {
```

Comment:
It would be good to add type hints to some of these functions while you're at it